### PR TITLE
For #117, switch to Jacoco for code coverage

### PR DESCRIPTION
**Pull request by pholser**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -255,6 +274,11 @@
                 <version>2.9</version>
             </plugin>
             <plugin>
+                <groupId>org.jacoco</groupId>
```
Comment (twoqubed) [inline pom.xml]: I don't see the report in the Maven site after running `mvn clean site:site`. Am I missing something?

Comment (pholser) [inline pom.xml]: Hm, yeah, I'm seeing this:

```
[INFO] Skipping JaCoCo execution due to missing execution data file:/Users/prholser/java/jopt-simple/target/jacoco-it.exec
```

Comment (pholser) [inline pom.xml]: NM - that's for the integration test execution.

Try `mvn clean package site:site`.

Comment (twoqubed) [inline pom.xml]: That worked. Thanks.

**Pull request merged**